src/Google/GlobalSiteTag.php

```diff
@@ -602,4 +606,118 @@ private function register_js_for_fast_refresh_dev() {
 			false
 		);
 	}
+
+	/**
+	 * Set user data config when Enhanced Conversions is enabled.
+	 *
+	 * @return string|null
+	 */
+	public function get_enhanced_conversion_tag() {
+		// TODO: Check enhanced conversion option status when implemented.
+		// $enhanced_conversions =$this->options->get( OptionsInterface::ADS_ENHANCED_CONVERSION_STATUS );
```

Comment:
✨

Reply:
@joemcgill I have synced this branch with the feature branch and updated to use the option.